---
title: Human genetics as a tool for drug discovery
author: 'frequena'
date: '2022-06-11'
slug: []
categories: [drug-discovery]
tags: []
subtitle: ''
summary: ''
authors: []
lastmod: '2022-07-11T13:07:40+02:00'
featured: no
image:
  caption: ''
  focal_point: ''
  preview_only: no
projects: []
---

For children with a rare disease, an accurate diagnosis is crucial to provide advice, possible therapies and assess the potential risk for family members in future generations. Public initiatives such as the International Rare Diseases Research Consortium (IRDiRC) set the goal for 2017-2027 to "enable all people living with a rare disease to receive an accurate diagnosis, care, and available therapy soon after seeking medical care" (1).

Despite significant efforts in the field of drug development, the reality is **discouraging**. Drug candidates that move from phase 1 clinical trials to approval and launch remain at around 10%. This rate is even lower in some specific fields, such as oncology (3.4%).

At the same time, large-scale biomedical databases, such as gnomAD and the UK Biobank, offer the possibility to assess the effect of variants in humans and their association with multiple conditions. In addition, the recruitment of population-based cohorts considered disease-agnostic increases the number of hypotheses that can be tested. In particular, the high scientific value of the UK Biobank is worth mentioning, as it not only releases genomic data but also rich phenotype characterization alongside other data sources (273).

Both mutations and drugs share the same mechanism: they disrupt the normal functioning of the human body. How they do so may differ, but it seems plausible to hypothesize that, in some cases, the phenotypic consequences of a loss-of-function SNVs or deletions are similar to the pharmacological effect of an inhibitor drug.

There is no doubt that this **view** poses some problems. As mentioned above, the heterogeneity and particularities in genetics are enormous even for monogenic diseases. Elements such as the genotype of the variant or discrepancies between the predicted effect and the actual consequences can make the use of this approach challenging. For instance, it is already known that LoF mutations may not decrease protein or even mRNA levels.

In spite of these drawbacks, **human genetics** seems to be a great tool for the identification of drugs with therapeutic effects.

There is evidence to support this assumption. For instance, gastrointestinal adverse events observed in clinical trials of DGAT1 inhibitors could have been predicted based on the causal relationship between rare and highly penetrant variants of DGAT1 and congenital diarrheal disorder (3).

Another well-known example is the association of heterozygous gain-of-function mutations in the PCSK9 gene and familial hypercholesterolemia, which leads to heart attacks or strokes relatively early in life. Strikingly, LoF variants in the PCSK9 gene have been causally associated with low levels of low-density lipoprotein cholesterol (4).

This **human genetic evidence** contributed to the technical and regulatory success of PCSK9 inhibitors, leading to the launch of evolocumab (Amgen) and alirocumab (Regeneron), and has also shown value in patient stratification in clinical trials (5).

Other drug candidates with strong genetic evidence between disease phenotypes and functional genetic variants have been identified, such as HSD17B13 for chronic liver disease (6), TYK2 for multiple autoimmune disorders (7), NRXN1 for neuropsychiatric disease (8), ASGR1 for cardiovascular disease (9),

These are not isolated examples, but a general trend. Nelson et al. found that pairing genetic target indication with genetic evidence almost doubles the success rate in clinical development. These analyses were re-evaluated three years later by other researchers with data after the original publication and controlling for potential confounding factors and corroborated the same claims (10).

Harnessing human genomic data can improve the drug discovery process, from target selection to reducing failures due to lack of efficacy or adverse effects.

## References

1.  .Sanders AD, Falconer E, Hills M, Spierings DCJ, Lansdorp PM. Single-cell template strand sequencing by Strand-seq enables the characterization of individual homologs. Nat Protoc. 2017;12: 1151--1176.
2.  Szustakowski JD, Balasubramanian S, Kvikstad E, Khalid S, Bronson PG, Sasson A, et al. Advancing human genetics research and drug discovery through exome sequencing of the UK Biobank. Nat Genet. 2021;53. <doi:10.1038/s41588-021-00885-0>
3.  Haas JT, Winter HS, Lim E, Kirby A, Blumenstiel B, DeFelice M, et al. DGAT1 mutation is linked to a congenital diarrheal disorder. J Clin Invest. 2012;122: 4680--4684.
4.  Cohen JC, Boerwinkle E, Mosley TH Jr, Hobbs HH. Sequence variations in PCSK9, low LDL, and protection against coronary heart disease. N Engl J Med. 2006;354: 1264--1272.
5.  Sabatine MS, Giugliano RP, Keech AC, Honarpour N, Wiviott SD, Murphy SA, et al. Evolocumab and Clinical Outcomes in Patients with Cardiovascular Disease. N Engl J Med. 2017;376: 1713--1722.
6.  Abul-Husn NS, Cheng X, Li AH, Xin Y, Schurmann C, Stevis P, et al. A Protein-Truncating HSD17B13 Variant and Protection from Chronic Liver Disease. N Engl J Med. 2018;378: 1096--1106.
7.  Dendrou CA, Cortes A, Shipman L, Evans HG, Attfield KE, Jostins L, et al. Resolving TYK2 locus genotype-to-phenotype differences in autoimmunity. Sci Transl Med. 2016;8: 363ra149.
8.  Noh HJ, Tang R, Flannick J, O'Dushlaine C, Swofford R, Howrigan D, et al. Integrating evolutionary and regulatory information with a multispecies approach implicates genes and pathways in obsessive-compulsive disorder. Nat Commun. 2017;8: 774.
9.  Nioi P, Sigurdsson A, Thorleifsson G, Helgason H, Agustsdottir AB, Norddahl GL, et al. Variant ASGR1 Associated with a Reduced Risk of Coronary Artery Disease. N Engl J Med. 2016;374: 2131--2141.
10. King EA, Davis JW, Degner JF. Are drug targets with genetic support twice as likely to be approved? Revised estimates of the impact of genetic support for drug mechanisms on the probability of drug approval. PLoS Genet. 2019;15: e1008489.
